packages/browser-wallet/src/popup/store/settings.ts

```diff
@@ -22,9 +23,13 @@ const storedNetworkConfigurationAtom = atomWithChromeStorage<NetworkConfiguratio
 );
 export const networkConfigurationAtom = atom<NetworkConfiguration, NetworkConfiguration>(
     (get) => get(storedNetworkConfigurationAtom),
-    (_, set, networkConfiguration) => {
+    (get, set, networkConfiguration) => {
         set(storedNetworkConfigurationAtom, networkConfiguration);
         popupMessageHandler.broadcast(EventType.ChainChanged, networkConfiguration.genesisHash);
+        const accounts = get(accountsAtom);
+        const identities = get(identitiesAtom);
+        set(selectedAccountAtom, accounts.length ? accounts[0] : undefined);
```

Comment:
I guess my previous comment is that this does not seem to work.

Reply:
Yeah, it used the list from the previous net. I've changed this, so it uses the correct net.